src/Components/Inquiry/Views/InquirySpecialist.tsx

```diff
@@ -59,7 +59,7 @@ export const InquirySpecialist: React.FC = () => {
         contactGallery: false,
       })
       setMode("Success")
-      await wait(500)
+      await wait(1500)
```

Comment:
What about just sending a toast?

Reply:
A toast is a great idea! I'll do that now :)